Library/Formula/h2o.rb

```diff
@@ -22,10 +22,9 @@ class H2o < Formula
 
   def install
     args = std_cmake_args
-    args << "."
     args << "-DWITH_BUNDLED_SSL=OFF"
 
-    system "cmake", *args
+    system "cmake", ".", *args
```

Comment:
Got rid of it in a new commit. No point keeping around redundant code. It would have bothered me if I'd left it in :smile_cat: